### DISCUSSION TITLE
Error when running native:build

### DISCUSSION THREAD

**Discussion by wcoy7** [NativePHP/laravel#227] [Q&A]
RollupError: Could not resolve "../../resources/php/php?asset&asarUnpack" from "src/main/index.js"

I can run native:serve no issues, but when I do the build command:

Could not resolve "../../resources/php/php?asset&asarUnpack" from "src/main/index.js"

Any ideas on getting past this step?

**Comment by simonhamp** [comment]
This feels like it should be raised as [an issue](https://github.com/nativephp/laravel/issues/new?assignees=&labels=bug&projects=&template=bug.yml&title=%5BBug%5D%3A+), rather than a discussion thread

**Comment by NBA707** [comment]
Not sure where it should go.   Not sure how to resolve this.   I don't have access to Issues.    Which repository should this go to?   I just did a composer update and the error still is not resolved.  In the resources directory there is no php directory

**Reply by simonhamp** [reply]
You can create a new issue [here](https://github.com/nativephp/laravel/issues/new?assignees=&labels=bug&projects=&template=bug.yml&title=%5BBug%5D%3A+)

**Comment by NBA707** [comment]
Sorry, I spaced it on the LINK that you sent originally.   It looks like you have to have an active apple developer account ot build an app.   Hopefully this requirement can be bypassed in the future when you are just experimenting and don’t actually plan on deploying the app but just running it locally.

Thank you,

Warren

**Reply by simonhamp** [reply]
>Hopefully this requirement can be bypassed in the future

You don't need an active Apple developer account to create production builds just for testing.

You only need it to sign and notarize so that your app can be distributed to other users. Your app simply won't work on another user's machine without signing and notarizing.

We won't be doing anything to attempt to bypass this requirement
